packages/create-ima-app/template/app/page/home/__tests__/HomeViewSpec.js

```diff
@@ -0,0 +1,47 @@
+import { initImaApp, clearImaApp } from '@ima/plugin-testing-integration';
+import cards from '../../../public/cards.json';
+
+describe('Home page', () => {
+  const response = {
+    body: cards,
+  };
+  let app;
+  let http;
+
+  beforeEach(async () => {
+    app = await initImaApp({
+      initBindApp: (ns, oc) => {
+        http = oc.get('$Http');
+
+        // Mock http.get method so the application
+        // wont make any external requests
+        // and return mocked response
+        http.get = jest.fn().mockReturnValue(Promise.resolve(response));
+      },
+    });
+
+    await app.oc.get('$Router').route('/');
+  });
+
+  afterEach(() => {
+    clearImaApp(app);
+  });
+
+  it('can render component', async () => {
+    expect(document.querySelectorAll('.cards')).toHaveLength(1);
+    expect(document.querySelectorAll('.card')).toHaveLength(cards.length);
+
+    const paragraphElements = document.querySelectorAll('.card p');
+    const titleElements = document.querySelectorAll('.card h3');
+
+    for (let i = 0; i < cards.length; i++) {
+      let paragraphText = paragraphElements[i].innerHTML.replace(
```

Comment:
Since we are not overriding this variable value, we can use `const` to be clear about it.

Reply:
Set.